### PR TITLE
OAEWDGT-147 - Update the OAE Builder docs

### PR DESCRIPTION
**Pull request by christianvuerings**
https://jira.sakaiproject.org/browse/OAEWDGT-147

### CODE REVIEW

File: app/views/sdk/fulldevsetup.html.erb [hunk 1]
```diff
@@ -52,9 +53,9 @@ bundle install --without mysql
 num_users_groups = 25
 update_ui = <span class="blue">false</span>
 
+<span class="grey">#If you'll be working on the front-end code, change the repository to your own 3akai-ux repo.</span>
```
Comment (croby) [inline app/views/sdk/fulldevsetup.html.erb]: It's not entirely clear what you mean by 'change the repository to your own 3akai-ux repo' -- I understand it, but I think you might want to provide an example (here and below)

Also, add a space after the `#` in all these comments

File: app/views/sdk/fulldevsetup.html.erb [hunk 2]
```diff
@@ -53,11 +53,26 @@ num_users_groups = 25
...
+                <h4>If you'll be working on the front-end code, change the repository to your own 3akai-ux repo.</h4>
+            </div>
+<pre class="black">
+ui = {<span class="blue">"path"</span> => <span class="green">"../3akai-ux"</span>, <span class="blue">"repository"</span> => <span class="green">"git@github.com:christianv/3akai-ux.git"</span>, <span class="blue">"branch"</span> => <span class="green">"master"</span>, <span class="blue">"localbranch"</span> => <span class="green">"master"</span>, <span class="blue">"remote"</span> => <span class="green">"origin"</span>}
+</pre>
+        </div>
+
+        <div class="navigationalitems_codesnippit_container navigationalitems_codesnippit_container_intended">
+            <div class="navigationalitems_codesnippit_header">
+                <h4>If you'll be working on the back-end code, change the repository to your own nakamura repository.</h4>
```
Comment (croby) [inline app/views/sdk/fulldevsetup.html.erb]: This is true for the clone stage, but after that i'm not sure that a user would want to have their settings.rb file look like this - as they'd want to pull down from sakaiproject. What do you think

File: app/views/sdk/fulldevsetup.html.erb [hunk 3]
```diff
@@ -53,11 +53,26 @@ num_users_groups = 25
...
 </pre>
         </div>
+        <div class="navigationalitems_codesnippit_container navigationalitems_codesnippit_container_intended">
+            <div class="navigationalitems_codesnippit_header">
+                <h4>If you'll be working on the front-end code, change the repository to your own 3akai-ux repo.</h4>
+            </div>
+<pre class="black">
+ui = {<span class="blue">"path"</span> => <span class="green">"../3akai-ux"</span>, <span class="blue">"repository"</span> => <span class="green">"git@github.com:christianv/3akai-ux.git"</span>, <span class="blue">"branch"</span> => <span class="green">"master"</span>, <span class="blue">"localbranch"</span> => <span class="green">"master"</span>, <span class="blue">"remote"</span> => <span class="green">"origin"</span>}
```
Comment (croby) [inline app/views/sdk/fulldevsetup.html.erb]: Here and below let's replace your username with `<your github username here>`

Comment (christianvuerings) [inline app/views/sdk/fulldevsetup.html.erb]: @croby Will do, I'll also change the color which should make it clearer that they need to change it.

File: app/views/sdk/fulldevsetup.html.erb (hunk 2 above)
Comment (christianvuerings) [inline app/views/sdk/fulldevsetup.html.erb]: You're right, this is definitely the case when they aren't doing any back-end work. If they do, they'll probably want to have an updated version (from sakaiproject). What would suggest here? Completely remove this block?

Comment (croby) [inline app/views/sdk/fulldevsetup.html.erb]: Yeah, I think so - let's remove it

Comment (christianvuerings) [inline app/views/sdk/fulldevsetup.html.erb]: @croby Should be ok now.

**Pull request merged**